src/card.css

```diff
@@ -0,0 +1,114 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
It isn't part of card block so move it to normalize.css

Reply:
done.